examples/EmbedExternalUI/EmbedExternalExampleUI.cpp

```diff
@@ -341,6 +342,10 @@ class EmbedExternalExampleUI : public UI
 
     void uiIdle() override
     {
+        if (isEmbed()) {
```

Comment:
this check must be only for macOS, on the other systems we need to poll to get our own custom events.

Reply:
that is true, as discussed ...

Reply:
then I was not clear enough. the event loop is not needed on macos or windows, but it is always needed on linux since there is no native event loop to hook into.
on x11 we must always handle our own events.

Reply:
nah you were clear enough. I agreed to that but then wrote that piece in the wrong place... now fixing that.